requirements.txt

```diff
@@ -3,4 +3,5 @@ supervision
 tqdm
 Pillow>=7.1.2
 PyYAML>=5.3.1
-click
\ No newline at end of file
+click
+scikit-learn
```

Comment:
Is this needed? I don't see it used and it's a really heavy dependency (plus this is going to have a merge conflict with #87)